function/pkg/docker/runtimes/runtimes.go

```diff
@@ -86,20 +86,20 @@ func ContainerImage(runtime types.Runtime) string {
 	switch runtime {
 	case types.Nodejs12:
 		return "eu.gcr.io/kyma-project/function-runtime-nodejs12:4bed80da"
-	case types.Nodejs10:
-		return "eu.gcr.io/kyma-project/function-runtime-nodejs10:4bed80da"
+	case types.Nodejs14:
+		return "eu.gcr.io/kyma-project/function-runtime-nodejs14:PR-11091"
 	case types.Python38:
 		return "eu.gcr.io/kyma-project/function-runtime-python38:4bed80da"
```

Comment:
Please update image tags for the python and nodejs12 runtimes too. It's better to be sure that everything is up to date and another thing is someone else would don't get why there are different tags